modules/oadp-usecase-backup-odf.adoc

```diff
@@ -0,0 +1,265 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-use-cases/oadp-usecase-backup-using-odf.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-usecase-backup_{context}"]
+= Backing up an application using {oadp-short} and {odf-short}
+
+In this use case, you back up an application by using {oadp-short} and store the backup in an object storage provided by {odf-first}.
+
+* You create a object bucket claim (OBC) to configure the backup storage location. You use {odf-short} to configure an Amazon S3-compatible object storage bucket. {odf-short} provides MultiCloud Object Gateway (NooBaa MCG) and Ceph Object Gateway, also known as RADOS Gateway (RGW), object storage service. In this use case, you use NooBaa MCG as the backup storage location.
+* You use the NooBaa MCG service with {oadp-short} by using the `aws` provider plugin. 
+* You configure the Data Protection Application (DPA) with the backup storage location (BSL). 
+* You create a backup Custom Resource (CR) and specify the application namespace to back up.
+* You create and verify the backup.
+
+.Prerequisites
+
+* You have installed the {oadp-short} Operator.
+* You have installed the {odf-short} Operator.
```

Comment:
```suggestion
* You installed the {odf-short} Operator.
```
not a merge blocker, but for the future, you don't need past perfect here (for minimalism)